### PR TITLE
add more basic elastic options.

### PR DESCRIPTION
**Pull request by taozix**

### CODE REVIEW

File: weed/command/scaffold.go [hunk 1]
```diff
@@ -176,7 +176,11 @@ database = "seaweedfs"
 
 [elastic7]
 enabled = false
-servers = "http://localhost:9200"
+servers = "http://localhost1:9200,http://localhost2:9200,http://localhost3:9200"
```
Comment (chrislusf) [inline weed/command/scaffold.go]: can this be a list? similar to line 152 in redis addresses.

Comment (taozix) [reply weed/command/scaffold.go]: updated.

File: weed/filer/elastic/v7/elastic_store.go [hunk 2]
```diff
@@ -73,6 +66,25 @@ func (store *ElasticStore) Initialize(configuration weed_util.Configuration, pre
...
+	if username != "" && password != "" {
+		options = append(options, elastic.SetBasicAuth(username, password))
+	}
+	options = append(options, elastic.SetSniff(configuration.GetBool(prefix+"sniff_enabled")))
+	options = append(options, elastic.SetHealthcheck(configuration.GetBool(prefix+"healthcheck_enabled")))
+	store.maxPageSize = configuration.GetInt(prefix + "index.max_result_window")
+	if store.maxPageSize <= 0 {
+		store.maxPageSize = 10000
+	}
+	glog.Infof("filer store elastic endpoints: %s.", servers)
```
Comment (chrislusf) [inline weed/filer/elastic/v7/elastic_store.go]: change `%s` to `%v`?

Comment (taozix) [reply weed/filer/elastic/v7/elastic_store.go]: nb.

File: weed/filer/elastic/v7/elastic_store.go [hunk 3]
```diff
@@ -73,6 +66,25 @@ func (store *ElasticStore) Initialize(configuration weed_util.Configuration, pre
 	}
 	return nil
 }
+
+func (store *ElasticStore) initialize(configuration weed_util.Configuration, prefix string) (options []elastic.ClientOptionFunc) {
```
Comment (chrislusf) [inline weed/filer/elastic/v7/elastic_store.go:70]: this PR is merged. but usually `Initialize()` handles configuration object, and `initialize()` do the actual initialization work. 

This PR has the opposite way. Maybe you can help to refactor this in a separate PR?

Comment (taozix) [reply weed/filer/elastic/v7/elastic_store.go:70]: ok.

**Pull request merged**